python/ray/data/_internal/logical/rules/split_read_output_blocks.py

```diff
@@ -35,8 +34,6 @@ def compute_additional_split_factor(
         size_based_splits = round(max(1, expected_block_size / target_max_block_size))
     else:
         size_based_splits = 1
-    if cur_additional_split_factor:
-        size_based_splits *= cur_additional_split_factor
```

Comment:
hmm, i feel like we still need to keep this logic and `cur_additional_split_factor` as a parameter in this function. @stephanie-wang as original author to confirm?

Reply:
yeah, this seems still useful.

Reply:
sounds good, I'll add it back